modules/claims_api/lib/bd/bd.rb

```diff
@@ -64,19 +64,19 @@ def upload(claim:, pdf_path:, doc_type: 'L122', file_number: nil, original_filen
     # @return {parameters, file}
     def generate_upload_body(claim:, doc_type:, pdf_path:, file_number: nil, original_filename: nil)
       payload = {}
-      veteran_name = "#{claim.auth_headers['va_eauth_firstName']}_#{claim.auth_headers['va_eauth_lastName']}"
+      auth_headers = claim.auth_headers
+
+      veteran_name = "#{auth_headers['va_eauth_firstName']}_#{auth_headers['va_eauth_lastName']}"
       file_name = generate_file_name(doc_type:, veteran_name:, claim_id: claim.evss_id, original_filename:)
 
-      data = {
-        data: {
-          systemName: 'VA.gov',
-          docType: doc_type,
-          claimId: claim.evss_id,
-          fileNumber: file_number || claim.auth_headers['va_eauth_birlsfilenumber'],
-          fileName: file_name,
-          trackedItemIds: []
-        }
-      }
+      # new method based on doc type
+      data = if doc_type == 'L023'
+               birls_file_num = auth_headers['va_eauth_birlsfilenumber'] || file_number
+               build_body(doc_type:, file_name:, claim_id:, file_number: birls_file_num)
+             elsif doc_type == 'L705'
+               build_body(doc_type:, file_name:, participant_id: nil, claim_id:, tracked_item_ids:, file_number: nil)
+             end
+
```

Comment:
@rockwellwindsor-va the `build_body` method should assist in your ticket. See also def on line 130.